libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_IP.c

```diff
@@ -1656,8 +1656,6 @@ uint8_t ucProtocol;
 		{
 			/* The size of the IP-header is larger than 20 bytes.
 			The extra space is used for IP-options. */
```

Comment:
1. It looks like we've removed `ipconfigIP_PASS_PACKETS_WITH_IP_OPTIONS`, so this is no longer configurable correct?
1. A `{` was removed from line 1660, but I don't see a corresponding `}` removed anywhere.  Did this code not compile previously?

Reply:
ad 1) `ipconfigIP_PASS_PACKETS_WITH_IP_OPTIONS` was not part of PR 2227, `ipconfigUDP_PASS_ZERO_CHECKSUM_PACKETS` neither.
Now I rebased this PR 2227 to master, so it includes both 2225, and 2226.

ad 2) when I introduced: `ipIPV4_VERSION_HEADER_LENGTH_MIN`/`MAX`, I painstakingly copied an `f( ipconfigIP_PASS_PACKETS_WITH_IP_OPTIONS != 0)`, which didn't have a matching `#endif`. Later I removed that from two PR's.

Reply:
I just updated `FreeRTOSIPConfigDefaults.h`, it still missed the changes from the 2 earlier PR's. Hopefully it is mergeable now.
Thanks, Hein